tools/data-api/README.md

```diff
@@ -0,0 +1,15 @@
+## PoC: Data API v2
```

Comment:
could we update this to match the others (a summary, that `serve` is the command which should be used right now, etc)